ci-operator/jobs/openshift/installer/openshift-installer-main-presubmits.yaml

```diff
@@ -2478,6 +2478,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-installer-main-altinfra-images
     rerun_command: /test altinfra-images
+    skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
```

Comment:
Do we need to set `always_run: false`?

Reply:
I just removed `skip_if_only_changed` to take care of the error in a less drastic way. Honestly, I this setting `always_run: false` should also be OK.

Reply:
Make sense to me :D We did discuss image builds are mandatory always so this way sounds good 👍🏼